### PR TITLE
Validation

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

File: schemas/webledger-validator.js [hunk 1]
```diff
@@ -1,27 +1,48 @@
...
 
 const proof = {
   title: '@context',
-  required: true,
+
+  // TODO: update this schema for properties of PoW, possibly tighten this
+  // up to allow only certain proof types: 'Ed25519Signature2018'
+  // 'RsaSignature2018', make use of `schemas.linkedDataSignature2018()`
+
+  // jsw is not required
```
Comment (dlongley) [inline schemas/webledger-validator.js]: `jws`

File: schemas/webledger-validator.js [hunk 2]
```diff
@@ -1,27 +1,48 @@
 /*!
  * Copyright (c) 2017-2018 Digital Bazaar, Inc. All rights reserved.
  */
+'use strict';
+
 const {constants} = require('bedrock').config;
 const {schemas} = require('bedrock-validation');
 
 const proof = {
   title: '@context',
```
Comment (davidlehn) [inline schemas/webledger-validator.js]: This title seems wrong?

File: schemas/webledger-validator.js [hunk 3]
```diff
@@ -1,27 +1,48 @@
...
-  }
+    created: schemas.w3cDateTime(),
+    jws: {
+      type: 'string'
+    },
+    type: {
+      type: 'string'
+    },
+  },
+  additionalProperties: false
```
Comment (davidlehn) [inline schemas/webledger-validator.js]: I see the TODO above.  Will this work as is for now without other properties?

File: schemas/webledger-validator.js [hunk 4]
```diff
@@ -1,27 +1,48 @@
...
   properties: {
-    type: schemas.url(),
     creator: schemas.url(),
-    created: schemas.w3cDateTime()
-  }
+    created: schemas.w3cDateTime(),
+    jws: {
+      type: 'string'
+    },
+    type: {
```
Comment (davidlehn) [inline schemas/webledger-validator.js]: Would schemas.jsonldType() be better or is that too complex here?

**Pull request merged**